wallets/provider-ledger/src/helpers.ts

```diff
@@ -0,0 +1,54 @@
+const ETHEREUM_CHAIN_ID = '0x1';
+
+export const ETH_BIP32_PATH = "44'/60'/0'/0/0";
+
+const ledgerErrorMessages: { [statusCode: number | string]: string } = {
+  21781: 'The device is locked',
+  25871: 'Related application is not ready on your device',
+  27013: 'Action denied by user',
+  INSUFFICIENT_FUNDS: 'Insufficient funds for transaction',
+};
+
+export function getLedgerInstance() {
+  // chainId put in instance because it is needed to be able to get chainId from provider
```

Comment:
```suggestion
  // Instances have a required property which is `chainId` and is using in swap execution. Here we are setting it as Ethereum always since we are supporting only eth for now.  
```